charts/generic-service/templates/ingress.yaml

```diff
@@ -12,13 +12,33 @@ metadata:
   labels:
     {{- include "generic-service.labels" . | nindent 4 }}
   annotations:
+    kubernetes.io/ingress.class: {{ .Values.ingress.class | default "nginx" | quote }}
```

Comment:
the default value here isn't required since https://github.com/ministryofjustice/hmpps-helm-charts/blob/main/charts/generic-service/values.yaml#L47 will set the default value as `nginx` if not specified anyway so can be
```
    kubernetes.io/ingress.class: {{ .Values.ingress.class }}
```

Reply:
I'm wondering now though - if there is only two values then perhaps should be in the `modsecurity_enabled` block instead - see https://github.com/ministryofjustice/hmpps-interventions-service/blob/83505bed995cf3486d6e17e8947c2168008cb277/helm_deploy/hmpps-interventions-service/templates/ingress.yaml